isort/stdlibs/py312.py

```diff
@@ -14,7 +14,9 @@
     "argparse",
     "array",
     "ast",
+    "asynchat",
```

Comment:
Please remove these.

They were all removed in 3.12:

https://docs.python.org/3/whatsnew/3.12.html#asynchat-and-asyncore

Reply:
I can remove it, but it was added by the scripts. So it will automatically appear on the next run because it's still present in the corresponding documentation: https://docs.python.org/3.12/library/asynchat.html#module-asynchat

Mb it would be better to use [stdlibs](https://github.com/omnilib/stdlibs) package or adopt it's approach to modules generation?

Reply:
Aha, the problem was caused by the docs re-adding module pages which had bee removed, but as stubs:

https://github.com/python/cpython/pull/126622

> Mb it would be better to use [stdlibs](https://github.com/omnilib/stdlibs?rgh-link-date=2024-12-27T18%3A09%3A16Z) package or adopt it's approach to modules generation?

Yeah, sounds like a good idea to use stdlibs 👍

Reply:
updated the script to use `stdlibs` and ran it